diff --git a/node_modules/eufy-security-client/build/http/types.js b/node_modules/eufy-security-client/build/http/types.js
index ea32f52..628ef65 100644
--- a/node_modules/eufy-security-client/build/http/types.js
+++ b/node_modules/eufy-security-client/build/http/types.js
@@ -9683,6 +9683,21 @@ exports.DeviceCommands = {
         CommandName.DeviceSavePresetPosition,
         CommandName.DeviceDeletePresetPosition,
     ],
+    [DeviceType.INDOOR_PT_CAMERA_E30]: [
+        CommandName.DeviceStartLivestream,
+        CommandName.DeviceStopLivestream,
+        CommandName.DeviceTriggerAlarmSound,
+        CommandName.DevicePanAndTilt,
+        CommandName.DeviceStartDownload,
+        CommandName.DeviceCancelDownload,
+        CommandName.DeviceCalibrate,
+        CommandName.DeviceStartTalkback,
+        CommandName.DeviceStopTalkback,
+        CommandName.DeviceSnooze,
+        CommandName.DevicePresetPosition,
+        CommandName.DeviceSavePresetPosition,
+        CommandName.DeviceDeletePresetPosition,
+    ],
     [DeviceType.INDOOR_PT_CAMERA_C210]: [
         CommandName.DeviceStartLivestream,
         CommandName.DeviceStopLivestream,
@@ -10236,6 +10251,16 @@ exports.StationCommands = {
         CommandName.StationDatabaseCountByDate,
         CommandName.StationDatabaseDelete,
     ],
+    [DeviceType.INDOOR_PT_CAMERA_E30]: [
+        CommandName.StationReboot,
+        CommandName.StationTriggerAlarmSound,
+        CommandName.StationDownloadImage,
+        CommandName.StationDatabaseQueryLatestInfo,
+        CommandName.StationDatabaseQueryByDate,
+        CommandName.StationDatabaseQueryLocal,
+        CommandName.StationDatabaseCountByDate,
+        CommandName.StationDatabaseDelete,
+    ],
     [DeviceType.INDOOR_PT_CAMERA_C210]: [
         CommandName.StationReboot,
         CommandName.StationTriggerAlarmSound,
